File: 1/Programazione/16 sett 24/es4.py
a=int(input("inserisci a: "))
b=int(input("inserisci b: "))
c=int(input("inserisci c: "))
d=int(input("inserisci d: "))
somma=a+b+c+d
media=int(somma)/4

if ((a-media)<=(b-media)):
    if ((a-media)<=(c-media)):
        if ((a-media)<=(d-media)):
            print("a")
        else:
            print("d")
    elif((c-media)<(d-media)):
        print("c")
    else:
        print("d")
else:
    if ((b-media)<=(c-media)):
        if ((b-media)<=(d-media)):
            print("b")
        else:
            print("d")
    elif((c-media)<(d-media)):
        print("c")
    else:
        print("d")

File: 1/Programazione/8_ott_24/es4.cpp
#include <iostream>
using namespace std;

int main(){
  int n, sum=0;
  printf("Inserisci n: \n");
  cin >> n;
  do{
    sum+=n%10;
    n=n/10;
  }
  while (n>0);
  cout << "Somma n: "<<sum<<endl;
}
